client/SDL/sdl_freerdp.cpp

```diff
@@ -305,16 +305,16 @@ static BOOL sdl_redraw(sdlContext* sdl)
 	return gdi_send_suppress_output(gdi, FALSE);
 }
 
-class EventGuard
+class SdlEventUpdateTriggerGuard
```

Comment:
Could this fixup commit be squashed into the first commit here?

Reply:
done